jwst/cube_build/cube_build_step.py

```diff
@@ -45,6 +45,7 @@ class CubeBuildStep (Step):
          single = boolean(default=false) # Internal pipeline option used by mrs_imatch and outlier detection
          output_type = option('band','channel','grating','multi',default='band') # Type of output cube to create. Options = band,channel,grating, multi
```

Comment:
I guess this comment line isn't quite long enough to cause trouble with parsing the params?